XRTK-Core/Packages/com.xrtk.core/Interfaces/InputSystem/Controllers/Hands/IMixedRealityHandRay.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace XRTK.Interfaces.InputSystem.Controllers.Hands
+{
+    /// <summary>
+    /// Hand ray types interface.
+    /// </summary>
+    public interface IMixedRealityHandRay
```

Comment:
I'd rather just have a `IMixedRealityRay` for now and then we can create a more specific type if required.